src/System.Runtime.CompilerServices.Unsafe/ver/Versioning.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0</TargetFrameworks>
     <AssemblyName>System.Runtime.CompilerServices.Unsafe</AssemblyName>
+    <IsSourceProject>false</IsSourceProject>

```

Comment:
Curious if this mattered, I was surprised to see it set here as I wouldn't expect the defaults to be wrong.

Reply:
I wanted to avoid setting the CLSCompliant property to true and also avoid adding metadata attributes to the output of Versioning.proj. I was being cautious as I am unsure if there could be unintended consequences. We already skip setting these elements in test projects, so it made sense to me to also skip setting it in this non-assembly project.